glue/qt/widgets/data_slice_widget.py

```diff
@@ -45,33 +46,54 @@ def __init__(self, label='', pix2world=None, lo=0, hi=10,
 
         layout.addLayout(top)
 
-        slider = QSlider(Qt.Horizontal)
-        slider.setMinimum(lo)
-        slider_lbl = QLabel()
-        slider.setMaximum(hi)
-        slider.setValue((lo + hi) / 2)
-        slider.valueChanged.connect(lambda x:
-                                    self.slice_changed.emit(self.mode))
-        slider.valueChanged.connect(lambda x: slider_lbl.setText(str(x)))
-        layout.addWidget(slider_lbl)
+        slider = load_ui('cube_slider')
+        slider.slider
+
+        slider.slider.setMinimum(lo)
+        slider.slider.setMaximum(hi)
+        slider.slider.setValue((lo + hi) / 2)
+        slider.slider.valueChanged.connect(lambda x:
```

Comment:
This was what you had originally, and it turns out that it was needed because ``self.mode`` isn't actually defined until ``__init__`` has run. Does this make sense?